PullRequestClosed/PullRequestClosed.test.ts

```diff
@@ -0,0 +1,112 @@
+import * as github from '@actions/github';
+import { PullRequestClosed } from './PullRequestClosed';
+import { LogTester } from '../tests/LogTester';
+import { jiraClientStub } from '../tests/JiraClientStub';
+import { createOctokitRestStub } from '../tests/OctokitRestStub';
+
+
+async function runAction(jiraProject: string, title: string, body?: string, user: string = 'test-user') {
+  process.env['INPUT_JIRA-PROJECT'] = jiraProject;
```

Comment:
It never changes => doesn't need to be parametrized
```suggestion
async function runAction(title: string, body?: string, user: string = 'test-user') {
  process.env['INPUT_JIRA-PROJECT'] = 'KEY';
```